tests/test_SequenceManager.cc

```diff
@@ -529,3 +532,265 @@ TEST_CASE("SequenceManager: store_sequence() & load_sequence() - Empty sequence"
         REQUIRE(seq_deserialized.empty());
     }
 }
+
+
+TEST_CASE("SequenceManager: git repository", "[SequenceManager]")
+{
+    std::filesystem::path git_dir {"sequences_git"};
```

Comment:
Let's not create another random folder with unit test files in the working directory. Instead, this should be a subfolder of `unit_test_files`.